### PR TITLE
[zh-cn] sync concepts/services-networking

### PR DESCRIPTION
**Pull request by my-git9**
[zh-cn] sync concepts/services-networking

### CODE REVIEW

File: content/zh-cn/docs/concepts/services-networking/service-traffic-policy.md [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
 title: 服务内部流量策略
 content_type: concept
-weight: 45
----
-<!-- 
+weight: 75
+description: >-
+  如果集群中的两个 Pod 想要通信，并且两个 Pod 实际上都在同一个节点上运行，
+  则**服务内部流量策略（Service Internal Traffic Policy）**将网络流量保持在该节点内。 
+  避免通过集群网络进行往返有助于提高可靠性、性能（网络延迟和吞吐量）或成本。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/service-traffic-policy.md:8]:
```suggestion
  避免通过集群网络进行往返有助于提高可靠性、增强性能（网络延迟和吞吐量）或降低成本。
```

File: content/zh-cn/docs/concepts/services-networking/topology-aware-hints.md [hunk 2]
```diff
@@ -1,16 +1,21 @@
 ---
 title: 拓扑感知提示
 content_type: concept
-weight: 45
----
-<!-- 
+weight: 70
+description: >-
+  **拓扑感知提示（Topology Aware Hints）**提供了一种机制来帮助将网络流量保持在其起源的区域内。
+  在集群中的 Pod 之间优先使用相同区域的流量有助于提高可靠性、性能（网络延迟和吞吐量）或成本。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/topology-aware-hints.md:7]:
```suggestion
  在集群中的 Pod 之间优先使用相同区域的流量有助于提高可靠性、增强性能（网络延迟和吞吐量）或降低成本。
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 3]
```diff
@@ -4,7 +4,8 @@ feature:
   title: 服务发现与负载均衡
   description: >
     无需修改你的应用程序即可使用陌生的服务发现机制。Kubernetes 为容器提供了自己的 IP 地址和一个 DNS 名称，并且可以在它们之间实现负载均衡。
-
+description: >-
+  将在集群中运行的应用程序暴露在单个外向端点后面，即使工作负载分散到多个后端也是如此。
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/services-networking/service.md:8]: Seems like `description` is duplicated.

File: content/zh-cn/docs/concepts/services-networking/service-traffic-policy.md [hunk 4]
```diff
@@ -1,16 +1,23 @@
 ---
 title: 服务内部流量策略
 content_type: concept
-weight: 45
----
-<!-- 
+weight: 75
+description: >-
+  如果集群中的两个 Pod 想要通信，并且两个 Pod 实际上都在同一个节点上运行，
+  则**服务内部流量策略（Service Internal Traffic Policy）**将网络流量保持在该节点内。
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/services-networking/service-traffic-policy.md:7]: [The preview](https://deploy-preview-37009--kubernetes-io-main-staging.netlify.app/zh-cn/docs/concepts/services-networking/) of `**bold**` doesn't seems right.

**Comment by howieyuen** [comment]
found a dup but more acceptable of:
```
content/zh-cn/docs/concepts/services-networking/service-traffic-policy.md
```
https://github.com/kubernetes/website/pull/37025

File: content/zh-cn/docs/concepts/services-networking/service.md (hunk 3 above)
Comment (my-git9) [reply content/zh-cn/docs/concepts/services-networking/service.md:8]: I will not modify this file first

File: content/zh-cn/docs/concepts/services-networking/topology-aware-hints.md [hunk 5]
```diff
@@ -1,16 +1,21 @@
 ---
 title: 拓扑感知提示
 content_type: concept
-weight: 45
----
-<!-- 
+weight: 70
+description: >-
+  **拓扑感知提示（Topology Aware Hints）**提供了一种机制来帮助将网络流量保持在其起源的区域内。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/topology-aware-hints.md:6]:
```suggestion
  **拓扑感知提示（Topology Aware Hints）**提供了一种机制来帮助将网络流量保持在其请求方所在的区域内。
```

File: content/zh-cn/docs/concepts/services-networking/topology-aware-hints.md [hunk 6]
```diff
@@ -1,16 +1,21 @@
...
+  在集群中的 Pod 之间优先使用相同区域的流量有助于提高可靠性、增强性能（网络延迟和吞吐量）或降低成本。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/topology-aware-hints.md:7]:
```suggestion
  在集群中的 Pod 之间优先选用相同区域的流量有助于提高可靠性、增强性能（网络延迟和吞吐量）或降低成本。
```

**Comment by Sea-n** [comment]
/retitle [zh-cn] Sync services-networking/topology-aware-hints.md

**Pull request merged**